eQuiz/modules/eQuiz.Web/Areas/Admin/Scripts/shared-values.service.js

```diff
@@ -0,0 +1,15 @@
+﻿(function (angular) {
+    angular.module('equizModule').service('sharedProperties', function () {

```

Comment:
Wrong syntax of filter declaration.
Use quizReview-data.service.js as an example